app/src/test/java/com/sebastianvm/mango/FakeProvider.kt

```diff
@@ -0,0 +1,24 @@
+package com.sebastianvm.mango
+
+import com.sebastianvm.mango.database.dao.FakeJobDao
+import com.sebastianvm.mango.database.models.JobEntity
+import kotlinx.coroutines.flow.MutableStateFlow
+import java.util.stream.Stream
+
+object FakeProvider {
+
+    val jobDao: FakeJobDao
+        get() = FakeJobDao(
```

Comment:
Using getter here because we can pair this with `@BeforeEach` to ensure that each test runs with a fresh instance of the Dao